config/index.js

```diff
@@ -2,5 +2,6 @@ module.exports = {
     server: {
         port: process.env.PORT || 5000
     },
-    mongodb_uri: process.env.MONGODB_URI
+    mongodb_uri: process.env.MONGODB_URI,
+    auth_token: process.env.AUTH_TOKEN
```

Comment:
Maybe you should add a check earlier in the file to ensure the auth_token is an ok value, or later you'll check against `undefined`